ext/src/CMakeLists.txt

```diff
@@ -2,4 +2,6 @@ if(WITH_ZPAGES)
   add_subdirectory(zpages)
 endif()
 
-add_subdirectory(http/client/curl)
+if(WITH_HTTP_CLIENT_CURL)
+  add_subdirectory(http/client/curl)
+endif()
```

Comment:
Not related to this PR, but while you are on it, can you add the copyright header here too :)

Reply:
Done.